packages/@n8n/ai-workflow-builder.ee/eslint.config.mjs

```diff
@@ -8,4 +8,9 @@ export default defineConfig(nodeConfig, {
 		'@typescript-eslint/require-await': 'warn',
 		'@typescript-eslint/naming-convention': 'warn',
 	},
+}, {
+	files: ['./src/test/**/*.ts', './**/*.test.ts'],
+	rules: {
+		'@typescript-eslint/no-unsafe-assignment': 'warn',
```

Comment:
fixes issue where `expect.any` and similar syntax marked as eslint errors. This is similar to other packages.